### ISSUE TITLE
My Devices tab - Anchoring/Sorting

### ISSUE THREAD

**Issue by fanynek86** [Ylianst/MeshCentral#6497]
Hello gentlemen!
I would like to add one small thing.
When scrolling through the list of devices, the top bar with the description does not stay docked, which is annoying, and also not all options like "Last Boot Up Time" or Windows Update are sorted.
![MC_My_Devices](https://github.com/user-attachments/assets/04f83214-4991-4780-a002-c0ecfbe222dd)

With Best Regards
František

**Comment by si458** [comment]
if you look at the old version of the web ui, its the same thing there
you cant sort by those columns, and the header bar doesnt stay there while you scroll
do you want these features adding to the new bootstrap version?

**Comment by fanynek86** [comment]
Yes, please... I checked both versions. I also noticed in the new version that if you select all the text in the filter field, you can delete it all with just the Delete key. If you use Backspace, only one character is deleted. I don't know if this is intentional or a bug.

**Comment by tobias9931** [comment]
not stale

**Comment by tobias9931** [comment]
not stale

**Comment by tobias9931** [comment]
not stale

**Comment by si458** [comment]
first part done (only year n half late haha) https://github.com/Ylianst/MeshCentral/commit/7a3f01db08a466de2de5c53b2de9d5884bd6554b

https://github.com/user-attachments/assets/91e8efc7-482d-4364-ba84-c5bccf5d427e

**Comment by si458** [comment]
added in last boot up time to sort as well for the moment, https://github.com/Ylianst/MeshCentral/commit/e8c53ffd59dd6b4e73a611cc40ecd8c7dd9a2122

<img width="2368" height="858" alt="Image" src="https://github.com/user-attachments/assets/64977ad6-470a-4a47-a03a-66278e7ecf6b" />

im going to hang fire on the other columns as we really need to rework the columns etc
